docs/site/home/community.html

```diff
@@ -285,7 +409,7 @@ <h2 class="accordion-header">
                         <h2 class="accordion-header" >
                             <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
                                     data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
-                                <img src="img/community/elixer.png" class="img-fluid me-3" alt="img">Elixer
+                                <img src="img/community/elixer.png" class="img-fluid me-3" alt="Elixir Logo">Elixer
```

Comment:
Elixer or Elixir?
also repository link in dead

Reply:
Thank you, @vkagamlyk! Updated.
Suggesting to update the repository to https://github.com/coingaming/gremlex since it looks like the most active one.